dashboard/src2/components/site/SitePerformance.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+	<div class="space-y-4">
+		<AlertBanner
+			title="All metrics presented are from the last 24 hours."
+			type="info"
+		/>
```

Comment:
ideally we shouldn't be showing this banner, instead we can give them docs like we have done in https://github.com/frappe/press/blob/7f7a26d1a19f57b8554578b18562967f89842c09/dashboard/src2/objects/tabs/patches.js#L21